pkgs/by-name/pr/pre-commit/package.nix

```diff
@@ -197,6 +197,10 @@ buildPythonApplication rec {
     "pre_commit"
   ];
 
+  preFixup = ''
+    makeWrapperArgs+=(--suffix PATH : ${lib.makeBinPath [ gitMinimal ]})
```

Comment:
Add a comment here saying this intentionally adds a fallback/low-prio git? (Or else someone might change it to --prefix or --set for purity.)